include/jsonv/optional.hpp

```diff
@@ -32,8 +32,13 @@
 #       undef  JSONV_OPTIONAL_USE_STD
 #       define JSONV_OPTIONAL_USE_STD 1
 #   else
-#       undef  JSONV_OPTIONAL_USE_STD_EXPERIMENTAL
-#       define JSONV_OPTIONAL_USE_STD_EXPERIMENTAL 1
+#       if defined(_MSC_VER)
+#           undef  JSONV_OPTIONAL_USE_BOOST
```

Comment:
Hi Travis , 
continuous-integration/appveyor/pr  - fails to build  due to the fact that 
Visual Studio does not have support for optional , we have no other choice, but to use boost::optional for msvc14. With c++17 visual studio will have std::optional and hopefully 
will fix __cplusplus , so your macro will work as is.
 
On the other hand I can try to modify 
C:\projects\json-voorhees\msvc\vs2015\jsonv-tests\jsonv-tests.vcxproj
so that integration build completes with success.
Let me know what you want to do with it.